src/app/parcels/ActionBar/ActionAndStatusBar.cy.tsx

```diff
@@ -156,7 +156,7 @@ describe("Parcels - Action Bar", () => {
             });
             cy.get(`input[value="${timeString}"]`).should("exist");
             mockData.forEach((parcel: ParcelsTableRow) => {
-                cy.get(".MuiPaper-root").contains(parcel.addressPostcode);
+                parcel.addressPostcode && cy.get(".MuiPaper-root").contains(parcel.addressPostcode);
```

Comment:
(same in other places)
```suggestion
                if (parcel.addressPostcode) {
                    cy.get(".MuiPaper-root").contains(parcel.addressPostcode);
                }
```